lib/rules/jsx-props-no-multi-spaces.js

```diff
@@ -98,8 +98,12 @@ module.exports = {
     }
 
     function containsGenericType(node) {
-      const containsTypeParams = typeof node.typeParameters !== 'undefined';
-      return containsTypeParams && node.typeParameters.type === 'TSTypeParameterInstantiation';
+      const nodeTypeParams = node.typeParameters;
+      if (typeof nodeTypeParams === 'undefined') {
+        return false;
+      }
+
+      return nodeTypeParams.type === 'TSTypeParameterInstantiation';
```

Comment:
it's strange that this line isn't covered by tests at all. any chance we could come up with a test case so this is covered?

Reply:
Locally and in CodeCov these lines are covered, I'm behind main and updated the brand, hopefully a rerun of the test suite will fix this.
Maybe the annotations are outdated, or I need to run the test suite on a specific Node.js version?

Reply:
if they're covered in codecov then it'll be fine, i'll rebase just in case